### DISCUSSION TITLE
Just activated but no stats yet

### DISCUSSION THREAD

**Discussion by Drallas** [athul/waka-readme#55] [Q&A] [answered]
I just created an account and activated an Action that runs fine but no data "The Data seems to be empty. Please wait for a day for the data to be filled in." 

Is it possible that even though i already have some stats in my https://wakatime.com/dashboard that i takes some time to populate?

**Comment by athul** [comment, marked as answer]
Did you make sure that you have the Wakatime API key in your repo secrets? The wakatime API only uopdates once a day, at Midnight. So if you have just made a wakatime account, just wait a day for the API to be updated

**Reply by Drallas** [reply]
ok thanks. i guess i wait until tomorrow, since i just setup a new Waka account and the Actions

i should have read the second part of the message "Please wait for a day for the data to be filled in"

**Reply by athul** [reply]
Yeah, if it doesn't work. Feel free to open an Issue.

**Reply by Drallas** [reply]
> Yeah, if it doesn't work. Feel free to open an Issue.

It worked, i just needed to be patience..

**Reply by ItzMarni** [reply]
Is there anyway to test the stats without waiting till midnight, i'm trying to figure out my looks for a profile but that would take litteral weeks before I got it 100% right because I need to test it every, and wait till midnight which is kinda bs...
